src/components/ProposalImagesPreview/ProposalImagesPreview.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+
+import Title from 'decentraland-gatsby/dist/components/Text/Title'
+import useFormatMessage from 'decentraland-gatsby/dist/hooks/useFormatMessage'
+
+import { ProposalAttributes } from '../../entities/Proposal/types'
+import ImageGallery from '../ImageGallery/ImageGallery'
+
+interface Props {
+  proposal: ProposalAttributes
+}
+
+function ProposalImagesPreview({ proposal }: Props) {
```

Comment:
I think this component can receive only the image_previews instead of the entire proposal 🤔